service/project.go

```diff
@@ -25,26 +25,45 @@ import (
 )
 
 // filterAuthorizedProjects iterates through a list of projects and returns a list of all the projects that a user
-// is authorized to view and edit the settings of.
+// is authorized to view and edit
 func (uis *UIServer) filterAuthorizedProjects(u gimlet.User) ([]model.ProjectRef, error) {
+	allProjects, err := model.FindAllProjectRefs()
```

Comment:
Do we need filterAuthorizedProjects anymore actually? It looks like the other places this is called just refreshes what the users should be able to view, which should probably also call filterViewableProjects